cli/deployment/deployment.go

```diff
@@ -0,0 +1,78 @@
+package deployment
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+)
+
+// Deployment object represent real GCP Deployment entity that already created or have to be created
+// configFile field point to yaml file, generated from Config object with all cross-deployment references
+// overwritten with actual values
```

Comment:
"The Deployment type represents a real GCP Deployment entity that is either already created, or yet-to-be created."

The bit about configFile should be attached to the `configFile` member, fixing YAML casing and as a complete sentence.